Kickstarter.xcodeproj/xcshareddata/xcschemes/Library-iOS.xcscheme

```diff
@@ -47,6 +47,16 @@
                ReferencedContainer = "container:Kickstarter.xcodeproj">
             </BuildableReference>
          </TestableReference>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "E113BD802B7D255000D3A809"
+               BuildableName = "Library-Keychain-iOSTests.xctest"
+               BlueprintName = "Library-Keychain-iOSTests"
+               ReferencedContainer = "container:Kickstarter.xcodeproj">
+            </BuildableReference>
+         </TestableReference>
```

Comment:
This schema change is so that when you test `Library-iOS`, it runs both `Library-iOSTests and Library-Keychain-iOSTests`

<img width="960" alt="Screenshot 2024-02-14 at 11 52 59 AM" src="https://github.com/kickstarter/ios-oss/assets/146007185/5495cbf7-133e-45d0-9f43-67c6ac9dedda">